### PR TITLE
Added pprof tool

### PR DESCRIPTION
**Pull request by d-ulyanov**
Added standard GO pprof tool to Telegraf. This tool is useful for developers: from time to time we need to understand, which plugin or subsystem of Telegraf consumes too much resources.

By default, pprof is disabled.
To enable pprof run Telegraf with `pprofaddr` parameter:
`telegraf --config etc/telegraf.conf --pprofaddr localhost:6060`

### CODE REVIEW

**Comment by sparrc** [comment]
thanks for adding this, can you add it to the `--help` doc? also put in the doc the format of the address, (ie, localhost:6060)

I would also call the flag `--pprof-addr` (with a dash)

**Comment by d-ulyanov** [comment]
@sparrc I've just added `pprof-addr` to help with example and renamed `pprofaddr` to `pprof-addr` - I agree, it seems better.

Reason why we've added pprof - high CPU consumption by Telegraf: up to 20% of total CPU on modern servers.
It was caused by built-in NETSTAT plugin - it makes too much system calls inside `github.com/shirou/gopsutil` to calculate network statistics. Please, consider replacement of this plugin.
As temporary solution - we've replaced netstat plugin by this: https://github.com/monitoring-tools/telegraf-plugins/tree/master/netstat

**Comment by d-ulyanov** [comment]
@sparrc please, run CI tests again: https://circleci.com/gh/influxdata/telegraf/6260 - its timed out

**Comment by d-ulyanov** [comment]
@sparrc Let's include this into 1.3.0?

File: cmd/telegraf/telegraf.go [hunk 1]
```diff
@@ -267,6 +275,18 @@ func main() {
 		processorFilters = strings.Split(":"+strings.TrimSpace(*fProcessorFilters)+":", ":")
 	}
 
+	if *pprofAddr != "" {
+		go func() {
+			log.Printf(
+				"I! Starting pprof on %s. Open profiling tools page in browser: /debug/pprof",
```
Comment (danielnelson) [inline cmd/telegraf/telegraf.go]: I think we have all the information we need to print the full url to this page, let have it write something like:
`I! Starting pprof HTTP server at http://localhost:6060/debug/pprof`  This way I can click on it or copy paste it more easily.

Comment (d-ulyanov) [inline cmd/telegraf/telegraf.go]: @danielnelson, thanks for review. Changes done.

Could you also move forward our contributions: 
https://github.com/influxdata/telegraf/pull/2387
https://github.com/influxdata/telegraf/pull/2594

We hope to see them in 1.3 too

**Comment by danielnelson** [comment]
Can you rebase?

**Comment by danielnelson** [comment]
Nevermind, It's merged!

**Pull request merged**